### PR TITLE
e2e - Automated test for application monitoring

### PR DESCRIPTION
**Pull request by howard-zhang**
e2e - Automated test for application monitoring

### CODE REVIEW

File: apps/tenant-management-webapp-e2e/src/integration/service-status.feature [hunk 1]
```diff
@@ -289,8 +289,29 @@ Feature: Service status
...
+  # When the user clicks Add notice button
+  # Then the user views Add notice dialog
+  # And no critical or serious accessibility issues for "add notice modal" on "status notices page"
+  # When the user clicks Cancel button in notice modal
+
+  @TEST_CS-2249 @REQ_CS-2058 @regression
+  Scenario: As a tenant admin, I can set a status application to be monitoring only, so it does not appear on the public status page
+    Given a tenant admin user is on status applications page
+    # Set monitor only for an application
+    When the user "selects" Monitor only checkbox for "autotest-testMonitoring"
```
Comment (LanaDIO) [inline apps/tenant-management-webapp-e2e/src/integration/service-status.feature:302]: I can not find "selects" Monitor checkbox, which I don't see in status application. Is it on Status service > Applications? or https://status.adsp-uat.alberta.ca/autotest

**Review by LanaDIO — COMMENTED** [review]
Unable to follow steps

File: apps/tenant-management-webapp-e2e/src/integration/service-status.feature (hunk 1 above)
Comment (howard-zhang) [reply apps/tenant-management-webapp-e2e/src/integration/service-status.feature:302]: It's in service-status in tenant management app. Status app is the public facing app for public to see the status of the applications.

Comment (LanaDIO) [reply apps/tenant-management-webapp-e2e/src/integration/service-status.feature:302]: Than is was I thought, in this case it should be status service>applications tab

**Comment by LanaDIO** [comment]
Than is was I thought, in this case it should be status service>applications tab

File: apps/tenant-management-webapp-e2e/src/integration/service-status.feature (hunk 1 above)
Comment (howard-zhang) [reply apps/tenant-management-webapp-e2e/src/integration/service-status.feature:302]: What tab? There is no step involving tabs. What's wrong with the following?

    Given a tenant admin user is on status applications page
    # Set monitor only for an application
    When the user "selects" Monitor only checkbox for "autotest-testMonitoring"

**Review by LanaDIO — APPROVED** [review]

**Pull request merged**